ocfweb/account/chpass.py

```diff
@@ -118,14 +123,16 @@ def change_password(request):
 
 class ChpassForm(Form):
 
-    def __init__(self, ocf_accounts, calnet_uid, *args, **kwargs):
+    def __init__(self, ocf_accounts: List[str], calnet_uid: str, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
         self.calnet_uid = calnet_uid
         self.fields['ocf_account'] = forms.ChoiceField(
             choices=[(x, x) for x in ocf_accounts],
             label='OCF account',
         )
-        self.fields.keyOrder = [
+
+        # mypy expects fields to be a dict, but it isn't. This is defined in django so it can't be fixed
+        self.fields.keyOrder = [  # type: ignore
```

Comment:
If you move this list into the class declaration like:
```
class ChpassForm(Form):
    field_order = [...]
```

you shouldn't need this type ignore.

c.f. https://stackoverflow.com/a/34502078